homeassistant/components/xiaomi_ble/sensor.py

```diff
@@ -129,13 +131,23 @@
         state_class=SensorStateClass.MEASUREMENT,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
-    # Used for e.g. consumable sensor on WX08ZM
-    (None, Units.PERCENTAGE): SensorEntityDescription(
-        key=str(Units.PERCENTAGE),
-        device_class=None,
+    # Used for e.g. consumable sensor on WX08ZM and M1S-T500
+    (ExtendedSensorDeviceClass.CONSUMABLE, Units.PERCENTAGE): SensorEntityDescription(
+        key=f"{ExtendedSensorDeviceClass.CONSUMABLE}",
         native_unit_of_measurement=PERCENTAGE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    # Used for score after brushing with a toothbrush
+    (ExtendedSensorDeviceClass.SCORE, None): SensorEntityDescription(
+        key=f"{ExtendedSensorDeviceClass.SCORE}",
```

Comment:
```suggestion
        key=str(ExtendedSensorDeviceClass.SCORE),
```

At some point we can make these true StrEnums in the upstream lib when we drop older python